com.nonpolynomial.intiface_central.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=Intiface Central
+Exec=run_intiface_central
+Type=Application
+Icon=com.nonpolynomial.intiface_central
```

Comment:
Missing categories here

Reply:
I cannot believe that was the solution for [this](https://github.com/flathub/flathub/pull/5012#issuecomment-1974293446).

How did you figure it out?

Reply:
It wasn't the new error was about missing categories. They weren't present in either places.

Reply:
Adding the categories in both the desktop file and metainfo solved the error though?

Reply:
Yea